[receiver/gitlab] add tracing via webhook skeleton (#36838)
open-telemetry/opentelemetry-collector-contrib: receiver/gitlabreceiver/README.md

# GitLab Receiver

## Traces - Getting Started

Workflow tracing support is actively being added to the GitLab receiver.
This is accomplished through the processing of GitLab webhook
events for pipelines. The [`pipeline`](https://docs.gitlab.com/ee/user/project/integrations/webhook_events.html#pipeline-events) event payloads are then constructed into `trace`
telemetry.

> Each GitLab pipeline, along with it's jobs, are converted

Reviewer: ```suggestion
Each GitLab pipeline, along with its jobs, is converted
```
